packages/Select/src/index.tsx

```diff
@@ -21,6 +21,7 @@ interface ISelectProps {
   validationMessage: string | JSX.Element;
   selected?: any;
   id?: string;
+  disabled?: any[],
```

Comment:
souldnt this be a boolean?

Reply:
no, it's per `option` tag. so you can disable more than one select's option